### DISCUSSION TITLE
comand Start-FinopsCostExport  without -Format

### DISCUSSION THREAD

**Discussion by vrondan1983** [microsoft/finops-toolkit#1055] [General]
Hello, im using powershell module FinopsTollkit , and its ok , but when we need to create  an export , does not exist the parameter "-Format" to select CSV or  Parquet , it  creates the export and it choose  by default CSV and i would like to use Parquet.

 I think it would be great if you add that option .

**Comment by flanakin** [comment]
Good call, we are missing this in New-FinOpsCostExport. Created #1074 to track this.
